dot/state/block_test.go

```diff
@@ -195,6 +195,62 @@ func TestGetSlotForBlock(t *testing.T) {
 	require.Equal(t, expectedSlot, res)
 }
 
+func TestGetBlockHashesBySlot(t *testing.T) {
```

Comment:
nit run it in parallel

```suggestion
func TestGetBlockHashesBySlot(t *testing.T) {
	t.Parallel()

```